charts/coredns/Chart.yaml

```
@@ -19,5 +19,5 @@ engine: gotpl
 type: application
 annotations:
   artifacthub.io/changes: |
-    - kind: Refactoring
-      description: Moved the labels from the the resource to a single helpers.tpl for better readability
+    - kind: Refactor
+      description: Update the github action.
```

See the artifacthub supported annotations and syntax at https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations. Using a supported key word makes the UI a bit nicer

```suggestion
    - kind: changed
      description: Update the github action.
```

---

` When using the list of objects option the valid supported kinds are added, changed, deprecated, removed, fixed, and security.`

I found the above so I have chosen the `Added`